### DISCUSSION TITLE
[Feature] immich-admin version command

### DISCUSSION THREAD

**Discussion by bo0tzz** [immich-app/immich#9611] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add a `version` subcommand to `immich-admin` that prints the `"version"` field from `package.json`.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
Implemented in #16924
